src/test/regress/expected/merge_unsupported_0.out

```diff
@@ -0,0 +1,106 @@
+SHOW server_version \gset
```

Comment:
This file is for PG15 and 16

Reply:
If we print the exact version then we need to separate the files for 15 and 16, I will fix it appropriately